R/clean_redcap.R

```diff
@@ -144,7 +145,10 @@ distill_nonrepeat_table <- function(
   # Below necessary to remove descriptive text fields
   # and to add column to indicate that form is completed
   my_fields <- db_data %>%
-    select(all_of(my_fields), paste0(my_form, "_complete")) %>%
+    select(all_of(my_fields),
+           matches(paste0("^",my_form, "_timestamp$")),
+           matches("^redcap_survey_identifier$"),
```

Comment:
Would this work? I think it better represents the intention of "these fields if they're in the data"
```suggestion
           anyof(c(paste0(my_form, "_timestamp"), "redcap_survey_identifier")),
```